index3.html

```diff
@@ -0,0 +1,285 @@
+              <input placeholder="R" type="text" id="r_exp" class="form-control" value="R">
+              <input placeholder="G" type="text" id="g_exp" class="form-control" value="G">
+              <input placeholder="B" type="text" id="b_exp" class="form-control" value="B">
+              <button type="submit" class="btn btn-primary">Run</button>
+            </form>
+
+            <form id="infragrammar_hsv" class="form-inline" style="display:none;" action="javascript:void(0);">
+              <input placeholder="H" type="text" id="h_exp" class="form-control" value="-(R-B)/(R+B)">
+              <input placeholder="S" type="text" id="s_exp" class="form-control" value="1">
+              <input placeholder="V" type="text" id="v_exp" class="form-control" value="1">
+              <button type="submit" class="btn btn-primary">Run</button>
+            </form>
+
+          </div>
+
+          <div class="zone colorize">
+            <p>3. <a href="https://publiclab.org/tag/colormap">COLORIZE</a></p>
+            <div class="btn-group">
+              <button type="submit" class="btn btn-lg btn-default btn-colorize"><i class="fa fa-adjust"></i></button>
+              <a class="btn btn-lg dropdown-toggle btn-default" data-toggle="dropdown" href="#">
+                <span class="caret"></span>
+              </a>
+              <ul class="dropdown-menu">
+                <li><a id="default_colormap">Default colormap</a></li>
+                <li><a id="stretched_colormap">Stretched colormap</a></li>
+              </ul>
+            </div>
+          </div>
+
+          <div class="zone">
+            <p>4. OVERLAY</p>
+            <div class="row">
+              <div class="col-md-4">
+                <button id="overlay-btn" class="btn btn-lg btn-default">
+                  <i class="fa fa-th" aria-hidden="true"></i>
+                </button>
+              </div>
+              <div class="col-md-8" id="overlay-controls-container" style="display:none;">
+                <input id="overlay-slider" type="range" min="10" max="50" value="100"/>
+                <span id="overlay-size" style="color:#ccc;"></span>
+                <button id="overlay-save-btn" class="btn btn-primary">Save</button>
+                <span id="overlay-save-info" style="color:green; display: none;">Size saved!</span>
+              </div>
+            </div>
+          </div>
+
+          <div class="zone last" id="save-zone" style="display:none;">
+            <p>OUTPUT</p>
+            <button onClick="openInPl()" type="button" class="btn btn-primary btn-lg">Save</button>
+            <button onClick="infragram.download()" type="button" class="btn btn-primary btn-lg">Download</button>
+            <button onClick="openInSequencer()" type="button" class="btn btn-primary btn-lg">Export</button>
+          </div>
+
+          <br style="clear:both;" />
+
+        </div>
+        <div id="localVideoControls" style="display:none;">
```

Comment:
So let's clearly label and explain these to @stephaniequintana who may be soon integrating this code into the new UI!